src/notebook/saving and loading.jl

```diff
@@ -110,7 +112,7 @@ function save_notebook(io::IO, notebook::Notebook)
     println(io, _cell_id_delimiter, "Cell order:")
     for c in notebook.cells
         delim = c.code_folded ? _order_delimiter_folded : _order_delimiter
-        println(io, delim, string(c.cell_id))
+        println(io, delim, string(c.cell_id) * (c.isolated ? _isolated_suffix : ""))
```

Comment:
If you use metadata for the isolated cell you can also probably avoid this additional delimiter for identifying isolated cells